packages/create-flex-plugin/templates/js/src/states/index.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers } from 'redux';
+
+import { reduce as CustomTaskListReducer } from './CustomTaskListState';
+
+// Register your redux store under a unique namespace
+export const namespace = '{{pluginNamespace}}';
+
+// Combine the reducers
+export default combineReducers({
+  customTaskList: CustomTaskListReducer
+});
```

Comment:
```
import { combineReducers } from 'redux';

 import { reduce as customTaskList } from './CustomTaskListState';

 // Register your redux store under a unique namespace
export const namespace = '{{pluginNamespace}}';

 // Combine the reducers
export default combineReducers({ customTaskList });
```

Reply:
When the file becomes longer, `customTaskList` is a confusing term. `CustomTaskListReducer` has the word `reducer` in it, which makes it more descriptive.